app/models/partner.rb

```diff
@@ -14,7 +14,7 @@
 class Partner < ApplicationRecord
   require "csv"
 
-  enum status: [:pending, :awaiting_review, :approved, :error]
+  enum status: [:uninvited, :invited, :awaiting_review, :approved, :error]
```

Comment:
there might be some consequences of adding these to the beginning of the list. The first one that comes to mind is we'll _probably_ need a data migration for existing partners that might be in `awaiting_review` but would now be `invited`.

Reply:
Great point! Also, all of the current partners who are pending (and have been invited) are going to show as uninvited!